sdk/examples/intkey_python/sawtooth_intkey/client_cli/intkey_cli.py

```diff
@@ -0,0 +1,354 @@
+        "%(white)s%(message)s",
+        datefmt="%H:%M:%S",
+        reset=True,
+        log_colors={
+            'DEBUG': 'cyan',
+            'INFO': 'green',
+            'WARNING': 'yellow',
+            'ERROR': 'red',
+            'CRITICAL': 'red',
+        })
+
+    clog.setFormatter(formatter)
+
+    if verbose_level == 0:
+        clog.setLevel(logging.WARN)
+    elif verbose_level == 1:
+        clog.setLevel(logging.INFO)
+    else:
+        clog.setLevel(logging.DEBUG)
+
+    return clog
+
+
+def setup_loggers(verbose_level):
+    logger = logging.getLogger()
+    logger.setLevel(logging.DEBUG)
+    logger.addHandler(create_console_handler(verbose_level))
+
+
+def create_parent_parser(prog_name):
+    parent_parser = argparse.ArgumentParser(prog=prog_name, add_help=False)
+    parent_parser.add_argument(
+        '-v', '--verbose',
+        action='count',
+        help='enable more verbose output')
+
+    try:
+        version = pkg_resources.get_distribution(DISTRIBUTION_NAME).version
+    except pkg_resources.DistributionNotFound:
+        version = 'UNKNOWN'
+
+    parent_parser.add_argument(
+        '-V', '--version',
+        action='version',
+        version=(DISTRIBUTION_NAME + ' (Hyperledger Sawtooth) version {}')
+        .format(version),
+        help='print version information')
+
+    return parent_parser
+
+
+def create_parser(prog_name):
+    parent_parser = create_parent_parser(prog_name)
+
+    parser = argparse.ArgumentParser(
+        parents=[parent_parser],
+        formatter_class=argparse.RawDescriptionHelpFormatter)
+
+    subparsers = parser.add_subparsers(title='subcommands', dest='command')
+
+    add_set_parser(subparsers, parent_parser)
+    add_inc_parser(subparsers, parent_parser)
+    add_dec_parser(subparsers, parent_parser)
+    add_show_parser(subparsers, parent_parser)
+    add_list_parser(subparsers, parent_parser)
+
+    add_generate_parser(subparsers, parent_parser)
+    add_load_parser(subparsers, parent_parser)
+    add_populate_parser(subparsers, parent_parser)
+    add_create_batch_parser(subparsers, parent_parser)
+    add_workload_parser(subparsers, parent_parser)
+
+    return parser
+
+
+def add_set_parser(subparsers, parent_parser):
+    parser = subparsers.add_parser('set', parents=[parent_parser])
+
+    parser.add_argument(
+        'name',
+        type=str,
+        help='the name of the value to set')
+
+    parser.add_argument(
+        'value',
+        type=int,
+        help='the value to set')
+
+    parser.add_argument(
+        '--url',
+        type=str,
+        help='the url of the REST API')
+
+    parser.add_argument(
+        '--keyfile',
+        type=str,
+        help='the file in which the user\'s private key is stored')
+
+    parser.add_argument(
+        '--wait',
+        nargs='?',
+        const=sys.maxsize,
+        type=int,
+        help='time to wait for transaction to commit')
```

Comment:
Need to specify the unit of time. I'm guessing that it's seconds. I prefer this version (similar to the rest_api wording):
     `Seconds to wait for the transaction to commit`

Also, there's some inconsistency across the CLIs - will this be standardized?
- seth has a short option as well as the long one (-w, --wait) 
- rest_api has a different name (-t TIMEOUT, --timeout TIMEOUT)